pkg/operators/kubeipresolver/kubeipresolver.go

```diff
@@ -28,6 +33,12 @@ import (
 
 const (
 	OperatorName = "KubeIPResolver"
+	Priority     = 10
+)
+
+const (
+	endpointL4Type = "gadget_l4endpoint_t"
+	ipAddrType     = "gadget_ip_addr_t"
```

Comment:
You mean setting annotations to allow operators to check if they should do something? Or what else do you have in mind?

Reply:
Yeah, exactly. Maybe the eBPF operator should be the only one knowing about types and operators work only on field annotations. But that discussion is out-of-scope for this PR. For now, let's go ahead like this.

Reply:
I found something similar on #3600. I created https://github.com/inspektor-gadget/inspektor-gadget/pull/3600/commits/c3b6bd861c045a543f8e2b33f2083d04723a8fdd. IMO it's fine for these operators to rely on the type:typename tag, otherwise how can we implement it?

Reply:
You're right. I got confused, it's friday 😅 . With that PR, everything is clearer.